.github/actions/e2e/create-cluster/action.yaml

```diff
@@ -125,6 +125,17 @@ runs:
         wellKnownPolicies:
           ebsCSIController: true
       EOF
+  - name: tag oidc provider of the cluster
```

Comment:
Can we make sure that we do an `always()` on this so that we always attempt to tag the OIDC even if the previous step doesn't succeed?

Reply:
That will give us a lower chance of leaking OIDC providers if something happens with the `eksctl create cluster`